README.md

```diff
@@ -1,5 +1,15 @@
 # cwgo
 
+## This repository has been archived.
+
+Due to the widespread use of Kitex Tool and Hertz by internal ByteDance users, and the lack of internal adoption scenarios for cwgo, we are unable to ensure sufficient code quality and manpower investment for new feature development. In the spirit of being responsible to our users, we have decided to archive the cwgo project. We sincerely thank everyone for their support and attention to the cwgo project.
```

Comment:
1. Kitex Tool and Hertz -> the built-in tools of Kitex and Hertz
2. thank everyone for their support... -> thank you all for your support ...